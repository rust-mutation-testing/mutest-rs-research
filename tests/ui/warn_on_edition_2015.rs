//@ print-mutants
//@ build
//@ stdout
//@ stderr
//@ edition: 2015

fn mutated_fn(a: i32, b: i32) -> i32 {
    a + b
}

#[test]
fn test() {
    assert_eq!(11, mutated_fn(5, 6));
}
