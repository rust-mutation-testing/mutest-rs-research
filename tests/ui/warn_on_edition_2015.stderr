warning: edition 2015 code is not explicitly supported
  |
  = note: you may encounter issues with generated code from macro expansion sanitization and mutation operators

warning: 1 warning emitted

